pdg/src/info.rs

```diff
@@ -259,25 +291,17 @@ mod test {
     /// ```
     #[test]
     fn unique_interleave() {
-        let mut g = Graph::default();
-
-        // let mut a = 0;
-        let a = mk_addr_of_local(&mut g, 0_u32);
-        // let b = &mut a;
-        let b1 = mk_copy(&mut g, a);
-        // *b = 0;
-        let b2 = mk_store_addr(&mut g, b1);
-        // let c = &mut a;
-        let c1 = mk_copy(&mut g, a);
-        // *c = 0;
-        let c2 = mk_store_addr(&mut g, c1);
-        // *b = 0;
-        let b3 = mk_store_addr(&mut g, b1);
-        // *c = 0;
-        let c3 = mk_store_addr(&mut g, c1);
-
-        let pdg = build_pdg(g);
-        check_unique(&pdg, &[], &[a, b1, b2, b3, c1, c2, c3]);
+        let mut g = CheckedGraph::default();
+
+        let a = g.mk_addr_of_local("let mut a = 0", false, 0_u32);
+        let b1 = g.mk_copy("let b = &mut a", false, a);
+        g.mk_store_addr("*b = 0", false, b1);
```

Comment:
How about `let _b2 = g.mk_store_addr(...)`, so the name `b2` is visible here for matching with the graph above?

Reply:
Sure, I can add them back and just `_`-prepend them.